src/mlpack/bindings/markdown/binding_info.cpp

```diff
@@ -23,9 +28,8 @@ util::ProgramDoc& BindingInfo::GetProgramDoc(const std::string& bindingName)
   return GetSingleton().map.at(bindingName);
 }
 
-/**
- * Register a ProgramDoc object with the given bindingName.
- */
+
+// Register a ProgramDoc object with the given bindingName.
```

Comment:
I think the comment style `//`` and ``/**/`` is somewhat interchangeable. But I don't have a strong opinion in this case, so I'm fine with the change.

Reply:
Likewise for me; however, Doxygen won't pick up the `//`, so we need to use `//!` instead.

Reply:
I will add ! @rcurtin sir.